src/main/resources/application.properties

```diff
@@ -8,7 +8,3 @@ elite2.api.uri.root=${elite2api.endpoint.url}/api
 oauth.endpoint.url=http://localhost:9090/auth
 casenotes.endpoint.url=http://localhost:8083
 api.health-timeout-ms=1000
-management.endpoints.web.base-path=/
```

Comment:
duplicate of application.yml (well it wasn't previously as was in the wrong place)